app/views/layouts/application.html.slim

```diff
@@ -1,6 +1,14 @@
 doctype html
 html[lang="en"]
   head
+    script[type="text/javascript"]
+      |  (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)}; m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)}) (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym"); ym(52661314, "init", { clickmap:true, trackLinks:true, accurateTrackBounce:true });
+    noscript
+      div
+        img[src="https://mc.yandex.ru/watch/52661314" style="position:absolute; left:-9999px;" alt=""]
+    script[async src="https://www.googletagmanager.com/gtag/js?id=UA-135652191-1"]
```

Comment:
do not commit keys) UA-135652191-1

Reply:
script[async src="https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_KEY"]}"]